tests/plugins/test_signer.py

```diff
@@ -162,141 +157,20 @@ async def test_signer_plugin_full_workflow(
         raise
 
 
-@pytest.mark.asyncio()
-async def test_workflow_without_review(
-    editor_client: RemoteSettingsClient,
-    reviewer_client: RemoteSettingsClient,
-    to_review_enabled: bool,
-):
```

Comment:
We have reviews enabled in dev currently and when running the docker compose images. Do we have any remote-settings environments with reviews disabled?

Reply:
Review is disabled on `dev`. Better said, dual signoff is disabled. That mean you can approve your own changes.
That's the only environment with this setting.

Then in local containers, we have the optional `testing.ini` config, which has the flag set too:

https://github.com/mozilla/remote-settings/blob/6169a51ec1b9ec04ead5198cdcfb9dc7b9cafb53/config/testing.ini#L71

We can have this approach for our tests:
- inspect the info on the root URL of the target server, to check whether review is disabled or not
- hardcode somewhere that if env==dev, then review has to be disabled otherwise has to be enabled

Reply:
Bringing `to_review_enabled` back in next commit, but pulling it from server config in a fixture. Also leaving an easy path to pull additional config values down the road.